### DISCUSSION TITLE
Can I blacklist a group of libraries/directories from being auto-included?

### DISCUSSION THREAD

**Discussion by Ma5t3rful** [clangd/clangd#1526] [Q&A]
Clang keeps auto including the following header to my projects: 

`#include <bits/chrono.h>`

this is fine in my system (Arch Linux) but on different Oses this will cause compile issues. (some distros doesn't have this header file and i have to remove `#include <bits/chrono.h>` manually and then it will work fine.)

is there any way to prevent auto-including this header? Im using vscode clangd extension

BTW, I know how to completely disable auto-including But having it is quite useful.

**Comment by HighCommander4** [comment]
> is there any way to prevent auto-including this header?

I'm not aware of a current way, though https://github.com/clangd/clangd/issues/1367 has a proposal which would include this.

That said, in the case of `<bits/chrono.h>` in particular, that sounds like a standard library implementation header, in which case clangd trying to include it is a bug that we should fix. Can you give an example of a symbol that triggers inclusion of this header, and also what standard library version you're using?
